src/main/scala/is/hail/methods/LDMatrix.scala

```diff
@@ -0,0 +1,58 @@
+package is.hail.methods
+
+import is.hail.utils._
+import is.hail.stats.ToNormalizedIndexedRowMatrix
+import is.hail.variant.{Variant, VariantDataset}
+import is.hail.utils.richUtils.RichIndexedRowMatrix._
+import org.apache.spark.mllib.linalg.distributed.{IndexedRow, IndexedRowMatrix}
+import org.apache.spark.storage.StorageLevel
+
+
+object LDMatrix {
+
+  /**
+    * Computes the LD matrix for the given VDS.
+    * @param vds VDS on which to compute Pearson correlation between pairs of variants.
+    * @return An LDMatrix.
+    */
+  def apply(vds : VariantDataset): LDMatrix = {
+    val nSamples = vds.nSamples
+    val originalVariants = vds.variants.collect().sorted
```

Comment:
I was originally skeptical that collect would preserve ordering, but it seems like it will on our OrderedRDD's, so I'm going to switch this to an assert.